app/cdash/tests/js/e2e_tests/calendar.js

```diff
@@ -1,6 +1,17 @@
 var LoginPage = require('../pages/login.page.js');
 const EC = protractor.ExpectedConditions;
 describe("calendar", function() {
+
+  beforeEach(async function() {
+    const browserLogs = await browser.manage().logs().get('browser');
+    browserLogs.forEach((log) => {
+      if (log.level.value > 900) { // it's an error log
+        console.log(`Browser console error: ${log.message}`);
+        fail(log.message);
+      }
+    });
+  });
```

Comment:
Would it not be better to run this after each test to potentially catch errors which occur during the test?